airbyte-integrations/connectors/source-gnews/source_gnews/gnews.yaml

```diff
@@ -44,7 +146,12 @@ definitions:
     nullable: "{{ ','.join(config['nullable']) }}"
     from: "{{ stream_slice['start_time'] }}"
     to: "{{ stream_slice['end_time'] }}"
+  schema_loader:
+    type: InlineSchemaLoader
   search_stream:
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
```

Comment:
fwiw I would find it more understandable if this said

```
  schema_loader:
    type: InlineSchemaLoader
```

instead of the ref

Reply:
Updated.